community/libcec/APKBUILD

```diff
@@ -1,54 +1,37 @@
-# Contributor: Carlo Landmeter <clandmeter@gmail.com>
+# Contributor: Róbert Nagy <vrnagy@gmail.com>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=libcec
-pkgver=2.2.0
-pkgrel=2
+pkgver=4.0.2
+pkgrel=0
 pkgdesc="libcec for the Pulse-Eight USB-CEC adapter"
 url="https://github.com/Pulse-Eight/libcec"
 arch="all"
 license="GPL"
-depends=""
-depends_dev="lockdev-dev eudev-dev"
-makedepends="$depends_dev autoconf automake libtool"
+depends="eudev p8-platform"
+depends_dev="eudev-dev p8-platform-dev python3-dev libxrandr-dev swig"
+makedepends="$depends_dev cmake automake"
 install=""
 subpackages="$pkgname-dev"
-source="http://github.com/Pulse-Eight/libcec/archive/libcec-$pkgver.tar.gz
-	musl-fix.patch
-	"
+source="http://github.com/Pulse-Eight/libcec/archive/libcec-$pkgver.tar.gz"
 
-_builddir="$srcdir"/libcec-libcec-$pkgver
-prepare() {
-	local i
-	cd "$_builddir"
-	for i in $source; do
-		case $i in
-		*.patch) msg $i; patch -p1 -i "$srcdir"/$i || return 1;;
-		esac
-	done
-        ./bootstrap || return 1
-}
+builddir="$srcdir"/libcec-libcec-$pkgver
 
 build() {
-	cd "$_builddir"
-	./configure \
-		--build=$CBUILD \
-		--host=$CHOST \
-		--prefix=/usr \
-		--sysconfdir=/etc \
-		--mandir=/usr/share/man \
-		--infodir=/usr/share/info \
-		--localstatedir=/var \
-		|| return 1
+	mkdir "$builddir/build"
+	cd "$builddir/build"
+	cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=1 -DCMAKE_INSTALL_PREFIX=/usr .. || return 1
+	make || return 1
 }
 
 package() {
-	cd "$_builddir"
-        make DESTDIR="$pkgdir" install || return 1
+	cd "$builddir/build"
+	make DESTDIR="$pkgdir" install || return 1
+
+	# Add cec directory to sys.path to get rid of issue https://github.com/Pulse-Eight/libcec/issues/316
+	python_dir=$(python3 -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())")
+	echo "cec" > "${pkgdir}${python_dir}/cec.pth"
```

Comment:
Without this line the module does not work. It's a bug in swig/python. Original issue: https://github.com/Pulse-Eight/libcec/issues/316